src/logic/operate.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    try {
+      return one.div(two).toString();
+    } catch (err) {
+      return "Can't divide by 0.";
+    }
+  }
+  if (operation === '%') {
+    try {
+      return one.mod(two).toString();
+    } catch (err) {
+      return "Can't find modulo as can't divide by 0.";
+    }
+  }
+  throw Error(`Unknown operation '${operation}'`);
+}
```

Comment:
while you did an impressive job working on the project requirements I noticed you disabled eslint . It is against micro verse policy to disable linters because linters help us write clean code. It would be nice if your enabled eslint and please do the same in all files were eslint was disabled.

Reply:
Thanks for the information. I will delete the dissabling line